package.json

```diff
@@ -39,6 +39,7 @@
     "express-session": "^1.15.6",
     "jsonwebtoken": "^8.4.0",
     "morgan": "^1.9.0",
+    "npm": "^6.7.0",
```

Comment:
What is this `npm` for?

Reply:
taking it out now, was trying to update node when that happen